cirq/google/api/v2/results.py

```diff
@@ -0,0 +1,174 @@
+from typing import (Dict, Iterable, Iterator, List, Optional, Set, Tuple, Union,
+                    cast)
+
+import numpy as np
+
+from cirq.api.google.v2 import result_pb2
+from cirq import circuits
+from cirq import devices
+from cirq import ops
+from cirq import schedules
+from cirq import study
+
+
+class Measurement:
```

Comment:
slight worry about the name....it will be only in Google namespace, but this could be confusing when people find "Measurement" here they might wonder how they use it in cirq core.

Reply:
I have the same hesitation, but couldn't think of a better name. Any suggestions?

Note that we could get rid of this entirely if `TrialResult` included the ids of the qubits that it applies to, which is probably a good idea anyway to make it more self-describing. I hope we can do that as part of the rework in #1689.

Reply:
Went with `MeasureInfo` for now, but happy to change that if we come up with a better name.